Spigot-Server-Patches/0511-Paper-dumpitem-command.patch

```diff
@@ -66,3 +66,15 @@ index a12bc81933c15606b7cde46937f504eafc4ff030..d798de637389cc5d5289a6c75304de30
      private void doFixLight(CommandSender sender, String[] args) {
          if (!(sender instanceof Player)) {
              sender.sendMessage("Only players can use this command");
+diff --git a/src/main/java/net/minecraft/server/NBTBase.java b/src/main/java/net/minecraft/server/NBTBase.java
+index 8b9e47b4c7f5dc464fa617a59583df9e30a54045..6ed1b4cd662c4b847a667e560b1e9a346c1baecb 100644
+--- a/src/main/java/net/minecraft/server/NBTBase.java
++++ b/src/main/java/net/minecraft/server/NBTBase.java
+@@ -24,6 +24,7 @@ public interface NBTBase {
+         return this.toString();
+     }
+ 
++    default IChatBaseComponent getPrettyNbt() { return this.l(); } // Paper - OBFHELPER
```

Comment:
minor thing, I would prolly name the obf helper to include that its a component, that it is intended for display

Reply:
Done